gui/src/ts/components/benchmark/BenchmarkPage.tsx

```diff
@@ -52,6 +52,11 @@ export const BenchmarkPage: React.FunctionComponent = () => {
   return (
     <Frame>
       <Grid container direction="column" spacing={6}>
+        <Grid item>
+          <Link to={Hrefs.benchmarks} data-cy="benchmarks-link">
```

Comment:
Joshua is working on a "BenchmarkFrame" that will have breadcrumbs and other common code for the benchmark page hierarchy, so this bespoke stuff will go away.

Reply:
nice